### DISCUSSION TITLE
Not support specify table name in table function when config.lower_case_table_names is not 0

### DISCUSSION THREAD

**Discussion by springMonkey** [apache/doris#16860] [A - General / Q&A]
SQL CONTENT:

`select 
	*
from (
	select
		M.DODE,
		M.ORDE,
		SUBSTR(M.DA_DATE, 1, 8) as DA_DATE,
		'releation_id' as RULE_ID,
		count(distinct(M.SID)) as TOTAL_ROWS,
		count(distinct(M.SID)) as UN_RELATE_ROWS,
		GROUP_CONCAT(CONCAT( M.SID, '|', M.IS_ERROR )) as SIDS
	from (
		select DODE , ORDE , DA_DATE , SID , IS_ERROR , CARDNO from 主表 where ( (DA_DATE >= '20221001000000' AND DA_DATE <= '20221001235959') ) 
	) M
	left join 
	(
		select '1' as SID, ORDE , CARDNO , DODE from 从表 where ( (DA_DATE > '20220924000000' AND DA_DATE <= '20221008235959') ) group by ORDE , CARDNO , DODE
	) S
	on M.ORDE = S.ORDE and M.CARDNO = S.CARDNO and M.DODE = S.DODE 
	group by M.DODE, M.ORDE, SUBSTR(M.DA_DATE, 1, 8)
) T 
lateral view explode_split(SIDS,",") TEMP1  as SID 	`

HOW TO SOLVETION THIS QUENTION?

**Comment by LemonLiTree** [comment]
Please record an issue first and describe the error message in detail
